components/server/src/workspace/workspace-service.ts

```diff
@@ -437,10 +437,21 @@ export class WorkspaceService {
         user: User,
         workspaceId: string,
         options: StartWorkspaceOptions = {},
+        restrictToRegular = true,
     ): Promise<StartWorkspaceResult> {
         await this.auth.checkPermissionOnWorkspace(user.id, "start", workspaceId);
 
-        const workspace = await this.doGetWorkspace(user.id, workspaceId);
+        const { workspace, latestInstance } = await this.getWorkspace(user.id, workspaceId);
+        if (latestInstance) {
+            if (latestInstance.status.phase !== "stopped") {
```

Comment:
@geropl I did not understand this change. It seems to change semantic?

Reply:
I tried to explain [here](https://github.com/gitpod-io/gitpod/pull/18653#discussion_r1314620049): This change is here to make this method en-par with GitpodServerImpl.startWorkspace.

It's not relevant for Prebuilds, bc. Prebuilds are always `Workspace` 1 <> 1 `WorkspaceInstance`.